spring-ai-openai/src/main/java/org/springframework/ai/openai/metadata/OpenAiRateLimit.java

```diff
@@ -33,20 +34,24 @@ public class OpenAiRateLimit implements RateLimit {
 
 	private static final String RATE_LIMIT_STRING = "{ @type: %1$s, requestsLimit: %2$s, requestsRemaining: %3$s, requestsReset: %4$s, tokensLimit: %5$s; tokensRemaining: %6$s; tokensReset: %7$s }";
 
+	@Nullable
```

Comment:
can this really be null?  at least in the case of openai it will always be returned I believe.